styles/_layout.less

```diff
@@ -10,3 +10,11 @@
 		padding-right: 1rem;
 	}
 }
+
+.begrensning--bred {
+	max-width: 820px;
+}
+
+.begrensning--hvit {
+	background-color: #fff;
```

Comment:
Hva er policy på bruk av variabler for farger?

Reply:
hvitt er hvitt :)